spec/acceptance/zabbix_application_spec.rb

```diff
@@ -5,12 +5,8 @@
 
 describe 'zabbix_application type', unless: default[:platform] =~ %r{archlinux} do
   supported_versions.each do |zabbix_version|
-    # 5.2 and 5.4 server packages are not available for RHEL 7
-    next if zabbix_version == '5.2' && default[:platform] == 'el-7-x86_64'
     # Application API was removed in Zabbix 5.4
     next if zabbix_version >= '5.4'
-    # No Zabbix 5.2 packages on Debian 11
-    next if zabbix_version == '5.2' && default[:platform] == 'debian-11-amd64'
```

Comment:
the case statement below has a block for version 4.

Reply:
so? 4.0 is LTS and I am not yet dropping it (well, in a month: https://www.zabbix.com/life_cycle_and_release_policy)

Reply:
ah okay